curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f356ed63c7807a4f1e6d054.md

```diff
@@ -6,13 +6,13 @@ dashedName: step-21
 ---
 
 # --description--
+The goal now is to make the `div` not take up the entire width of the page. The CSS `width` property is perfect for this.
```

Comment:
```suggestion

The goal now is to make the `div` not take up the entire width of the page. The CSS `width` property is perfect for this.
```

I'm guessing we want to keep the space between this text as the description heading above.